targeting/geoip2/geoip2.go

```diff
@@ -205,7 +268,7 @@ func (g *GeoIP2) HasLocation() (bool, error) {
 
 // GetLocation returns a geo.Location object for the given IP
 func (g *GeoIP2) GetLocation(ip net.IP) (l *geo.Location, err error) {
```

Comment:
don't we need a read lock here (and the other accessors) to avoid a race condition with the update? You lock the update, but if the reads aren't looking for the lock it doesn't matter, right?

Reply:
Updated on 7279695fa1bfec205671c456e9bd855c0b8bb0b3

Reply:
thanks! I didn't realize I already had a mutex partially implemented. I always meant to make the change you were doing, but didn't because I was concerned about adding the mutex and the potential performance impact (or making bugs). Duh...